src/spikeinterface/extractors/cbin_ibl.py

```diff
@@ -129,7 +129,7 @@ def get_traces(self, start_frame, end_frame, channel_indices):
         if not self._load_sync_channel:
             traces = traces[:, :-1]
 
-        return traces
+        return traces[:, channel_indices]
```

Comment:
You nede to check that this is not None othewise it increase the dimenssionility.